### ISSUE TITLE
Heat stack should add wildcard A record to zone

### ISSUE THREAD

**Issue by markllama** [redhat-openstack/openshift-on-openstack#330]
During the deployment with the 'loadbalancer_dedicated' option and dynamic DNS, the loadbalancer entry is created for the developers as <stack name>-devs.<zone>.  A comparable A record should be added for the applications: *.apps.<zone>.

Then when the zone is properly delegated users will be able to find the apps correctly.

**Comment by ioggstream** [comment]
Could this be implemented  for LBaaS too triggering a SoftwareDeployment on the bastion node?

**Comment by markllama** [comment]
Yes, though the DNS service has been moved out of the heat stack to reflect more real-world deployments.  The mechanism is different but the desired result is the same.

**Comment by ioggstream** [comment]
@markllama 
  if you have a LoadBalancer, openshift-on-openstack configure the nodes to point the loadbalancer dns record. This means you need either to:

  * configure the DNS as soon as openstack gives you the loadbalancer ip
  
or

  * setting a static ip for the load balancer, but I can't find a way to do it via the templates

Do you have any hint? 

Thx++,
R:
